vpr/CMakeLists.txt

```diff
@@ -10,6 +10,8 @@ option(VPR_USE_SIGNAL_HANDLER "Should VPR use a signal handler to intercept sign
 set(VPR_PGO_CONFIG "none" CACHE STRING "Configure VPR Profile-Guided Optimization (PGO). prof_gen: built executable will produce profiling info, prof_use: built executable will be optimized based on generated profiling info, none: disable pgo")
 set_property(CACHE VPR_PGO_CONFIG PROPERTY STRINGS prof_gen prof_use none)
 
+set(VPR_PGO_DATA_DIR "." CACHE PATH "Where to store and retrieve PGO data")
```

Comment:
I believe this should be "${CMAKE_CURRENT_SOURCE_DIR}"?

Reply:
The default is to put the profile data in the same place as the object files; one for each .o file, which requires a relative path. "." is the default if unspecified.